app/controllers/orders_controller.rb

```diff
@@ -23,10 +23,9 @@ def update
 
 
   def new
-    order = Order.create(status: "ordered", user_id: current_user.id)
-    item_hash = @cart.cart_items
-    order.add(item_hash)
+    OrderCreationService.create_from_cart(@cart.cart_items, current_user)
```

Comment:
Implementation of OrderCreationService PORO looks good in pulling out all of the logic involved in creating potentially multiple orders for a single transaction.